### PR TITLE
Fix/del rot smid rmid tests

### PR DESCRIPTION
**Pull request by 2byrds**
Catches KERIA up with KERIpy changes AND provides all functionality necessary for SIGNIFY-TS integration tests to pass, especially multisig-vlei-issuance but also fixes for rotation, salty, and delegation

### CODE REVIEW

File: src/keria/app/delegating.py [hunk 1]
```diff
@@ -119,6 +119,7 @@ def processEscrows(self):
     def processUnanchoredEscrow(self):
         """
         Process escrow of unacnchored events that have been delegated and are waiting for delegator anchor/approval.
+        Process escrow of unacnchored events that have been delegated and are waiting for delegator anchor/approval.
```
Comment (m00sey) [inline src/keria/app/delegating.py:122]: duplicated comment?

**Comment by 2byrds** [comment]
There is a small amount of overlap with https://github.com/WebOfTrust/signify-ts/pull/234/files but we'll adjust once either is merged.

File: src/keria/app/aiding.py [hunk 2]
```diff
@@ -483,9 +483,9 @@ def on_post(req, rep):
 
                 states = httping.getRequiredParam(body, "smids")
                 rstates = httping.getRequiredParam(body, "rmids")
-                smids = [state['i'] for state in states]
-                rmids = [rstate['i'] for rstate in rstates]
-                hab = agent.hby.makeSignifyGroupHab(name, mhab=mhab, smids=smids, rmids=rmids, serder=serder,
+                # smids = [state['i'] for state in states]
```
Comment (pfeairheller) [inline src/keria/app/aiding.py:486]: Please delete this code if its no longer needed.

File: src/keria/app/delegating.py [hunk 3]
```diff
@@ -226,23 +227,23 @@ def on_post(self, req, rep, name):
         except (kering.AuthError, ValueError) as e:
             raise falcon.HTTPBadRequest(description=e.args[0])
     
-    @staticmethod
-    def approveDelegation(agent, hab, anc) -> str:
-        serder = serdering.SerderKERI(sad=anc)
+@staticmethod
```
Comment (pfeairheller) [inline src/keria/app/delegating.py:230]: If this method is no longer a part of the class, this annotation is inappropriate

File: src/keria/app/aiding.py [hunk 4]
```diff
@@ -1914,11 +1914,15 @@ def on_get(req, rep, name):
 
         signing = []
         for smid in smids:
+            if isinstance(smid,dict):
+                smid = smid['i']
```
Comment (pfeairheller) [inline src/keria/app/aiding.py:1918]: Are there really cases where the contents of smids could be either a dict or a str?  We should figure out why and if that's necessary

File: src/keria/app/aiding.py [hunk 5]
```diff
@@ -1914,11 +1914,15 @@ def on_get(req, rep, name):
...
             ends = hab.endsFor(smid)
             signing.append(dict(aid=smid, ends=ends))
 
         rotation = []
         for rmid in rmids:
+            if isinstance(rmid,dict):
```
Comment (pfeairheller) [inline src/keria/app/aiding.py:1924]: Same as above

**Review by pfeairheller — CHANGES_REQUESTED** [review]

**Comment by 2byrds** [comment]
@pfeairheller I have made all the updates based on your review. I also added a small change to grouping.py to address @daviddm [issue](https://github.com/WebOfTrust/signify-ts/issues/227) and corresponding PR which contains an additional signify-ts test (https://github.com/WebOfTrust/signify-ts/pull/258) that will now pass and can be merged after this.

I also ran against all original singify-ts integration tests again:
image

**Review by pfeairheller — APPROVED** [review]

**Pull request merged**